template.yaml

```diff
@@ -1297,6 +1297,59 @@ Resources:
           EVENT_BUS: !GetAtt ImportRequestsEventBus.Name
           IMPORT_EVENT_TYPE: import.cristin.entry-event
 
+  #============================ Deploy API =============================================================
```

Comment:
Add a comment why is this necessary. It can be as simple as "This  solves the problem described here: http://....."